src/controller/org.controller/org.controller.js

```diff
@@ -427,7 +449,7 @@ async function createUser (req, res) {
 
   if (!orgUUID) {
     logger.info('The user could not be created because ' + cnaShortName + ' CNA does not exist.')
-    return res.status(400).json({ message: 'The user could not be created because ' + cnaShortName + ' CNA does not exist.' })
+    return res.status(404).json({ message: 'The user could not be created because ' + cnaShortName + ' CNA does not exist.' })
```

Comment:
I'm pretty sure this should stay a 400 because they aren't attempting to get the org or user, they're trying to create one and they provided the wrong org